frontend/packages/dev-console/src/components/monitoring/dashboard/MonitoringDashboard.tsx

```diff
@@ -1,12 +1,72 @@
 import * as React from 'react';
+import * as _ from 'lodash';
+import { match as RMatch } from 'react-router-dom';
 import { Helmet } from 'react-helmet';
+import { Bullseye, Grid, GridItem } from '@patternfly/react-core';
+import MonitoringDashboardGraph from './MonitoringDashboardGraph';
+
+export interface MonitoringDashboardProps {
+  match: RMatch<{
+    ns?: string;
+  }>;
+}
+const queries = [
+  {
+    query: _.template(
+      `topk(25, sort_desc(sum(avg_over_time(pod:container_cpu_usage:sum{container="",pod!="",namespace='<%= namespace %>'}[5m])) BY (pod, namespace)))`,
+    ),
+    chartType: 'area',
+    title: 'stat 1',
+  },
+  {
+    query: _.template(
+      `topk(25, sort_desc(sum(avg_over_time(container_memory_working_set_bytes{container="",pod!="",namespace='<%= namespace %>'}[5m])) BY (pod, namespace)))`,
+    ),
+    chartType: 'area',
+    title: 'stat 2',
+  },
+  {
+    query: _.template(
+      `topk(25, sort_desc(sum(pod:container_fs_usage_bytes:sum{container="",pod!="",namespace='<%= namespace %>'}) BY (pod, namespace)))`,
+    ),
+    chartType: 'line',
```

Comment:
can we have a separate file with predefined queries?

Reply:
Done.